kubernetes/samples/charts/traefik/README.md

```diff
@@ -91,6 +91,80 @@ TLS_PORT=`kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.po
 $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:${TLS_PORT}/testwebapp/
 ```
 
+## SSL Termination at Ingress Controller
+This sample demonstrates how to terminate SSL trafik at ingress contoller to access WebLogic console thru SSL port. 
+
+### 1. Enable "WebLogic Plugin Enabled" on WebLogic Domain level
```

Comment:
on WebLogic Domain level -> on the WebLogic domain.

Reply:
done